types/three/index.d.ts

```diff
@@ -29,5 +29,7 @@ export * from "./three-trackballcontrols";
 export * from "./three-transformcontrols";
 export * from "./three-vrcontrols";
 export * from "./three-vreffect";
+export * from "./three-OutlinePass";
+export * from "./three-SMAAPass";
```

Comment:
Please keep these alphabetically ordered. On that note, they're almost entirely lowercase and we should match that too.

Reply:
ok